patches/api/0373-Allow-usage-of-MiniMessage-in-server-configuration.patch

```diff
@@ -0,0 +1,73 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Peter Crawley <peterzcrawley@gmail.com>
+Date: Sat, 5 Mar 2022 19:08:41 +0000
+Subject: [PATCH] Allow usage of MiniMessage in server configuration files
+
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index 944f9b87a11472ac6d7e328acc00bf09f899e648..898f8c777cedee0752af4eb5743565498748cd1b 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -2226,13 +2226,24 @@ public final class Bukkit {
+ 
+     /**
+      *
++     * @deprecated Use org.bukkit.Bukkit#permissionMessage()
+      * @return the default no permission message used on the server
+      */
+     @NotNull
++    @Deprecated
+     public static String getPermissionMessage() {
+         return server.getPermissionMessage();
+     }
+ 
++    /**
++     *
++     * @return the default no permission message used on the server
++     */
++    @NotNull
++    public static net.kyori.adventure.text.Component permissionMessage() {
++        return server.permissionMessage();
++    }
++
+     /**
+      * Creates a PlayerProfile for the specified uuid, with name as null
+      * @param uuid UUID to create profile for
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index cca362e54d6ff4a5a1e60f85a7eb1b3d222d3d48..ae40fd36b25ce63ffd191fe76f1295f6fd9fb6dc 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -1935,11 +1935,20 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
+ 
+     /**
+      *
++     * @deprecated Use org.bukkit.Server#permissionMessage()
+      * @return the default no permission message used on the server
+      */
+     @NotNull
++    @Deprecated
+     String getPermissionMessage();
+ 
++    /**
++     *
++     * @return the default no permission message used on the server
++     */
++    @NotNull
++    net.kyori.adventure.text.Component permissionMessage();
++
```

Comment:
I think this change (and the similar ones) should be made in the patch that added them.

Reply:
Moved.